config/config.exs

```diff
@@ -5,7 +5,7 @@ config :logger, :console,
   format: "\n$time $metadata[$level] $message\n"
 
 config :phoenix,
-  json_library: Jason,
```

Comment:
Probably here we would need to have something similar to `if Code.ensure_loaded?(JSON), do: JSON, else: Jason` if that conditional loading trick makes sense.